src/features/page_view_timing/aggregate/index.js

```diff
@@ -43,6 +45,12 @@ export class Aggregate extends AggregateBase {
     timeToFirstByte.subscribe(({ entries }) => {
       this.addTiming('load', Math.round(entries[0].loadEventEnd))
     })
+    // *cli Mar'24 - CLS node won't be added until we fix the rounding problem in schema that's grounding the decimal value to 0
+    // subscribeToVisibilityChange(() => {
+    //   const { name, value, attrs } = cumulativeLayoutShift.current
+    //   if (value === undefined) return
+    //   this.addTiming(name, value * 1000, attrs) // downstream consumer interprets the value as ms-unit and converts it to seconds; cls score is neither and we need to negate that division
+    // }, true) // so CLS node only reports on vis change rather than on every change
```

Comment:
Are we OK with leaving commented out code in the repo?

Reply:
I'm expecting the fix needed from consumer to be done in the near future, I think. Just saves me later from re-writing/rethinking out what needs to be written lol.

Reply:
Along these lines, should the [CLS](https://github.com/newrelic/newrelic-browser-agent/pull/919/files#diff-c3fae4e9a454f8ce96b46ab1b4c0ec1cf02d4f5a52418979046ed371e4965d96R9-R17) change in the emitter also be removed/commented out since there isnt anything consuming it?

Reply:
Harmless to leave in really; the attrs just won't be consumed at all. I could comment out the attrs though if you feel like we should--only the value is needed/used atm with it as an attr under other nodes. So?